common/djangoapps/third_party_auth/tests/test_admin.py

```diff
@@ -66,6 +66,7 @@ def test_oauth2_provider_edit_icon_image(self):
         # Remove the icon_image from the POST data, to simulate unchanged icon_image
         post_data = models.model_to_dict(provider1)
         del post_data['icon_image']
+        del post_data['max_session_length']
```

Comment:
Will add a comment: long and the short of it is that the results of a nullable, blankable integer field with an empty value being serialized by `model_to_dict` are not isomorphic with what that value needs to look like to be POSTed back; `application/x-www-form-urlencoded` has no concept of a null key.